searchProc.js

```diff
@@ -0,0 +1,122 @@
+let storeName, storeNumber, roadName, menu, storeX, storeY, id, writer;
+let num = 1;
+
+//모든 리스트를 보여주는 함수
+function getData() {
+    console.log('getData!!!!!!')
+    const xhr = new XMLHttpRequest();
+    xhr.open('GET', '/api/storeDB', true);
+    xhr.onreadystatechange = function () { // Call a function when the state changes.
+        if (this.readyState === XMLHttpRequest.DONE && this.status === 200) {
+            // Request finished. Do processing here.
+            const responseData = JSON.parse(this.response);
+            console.log(responseData);
+            //list를 출력하는 for
+            for (var i = 0; i <= responseData.length; i++) {
```

Comment:
0부터 시작하니 배열의 길이보다 작은 인덱스까지 루프를 돌려야 할것 같네요.
```suggestion_
            for (var i = 0; i < responseData.length; i++) {
```